deployment/helm/node-feature-discovery/templates/worker.yaml

```diff
@@ -43,6 +43,17 @@ spec:
           {{- toYaml .Values.worker.securityContext | nindent 12 }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        livenessProbe:
+          grpc:
+            port: 8082
+          initialDelaySeconds: 10
+          periodSeconds: 10
+        readinessProbe:
+          grpc:
+            port: 8082
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          failureThreshold: 10
```

Comment:
since you added here, probably we could it in https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/deployment/helm/node-feature-discovery/values.yaml as well but have it commented out for the sake of extra documentation. Similar to https://github.com/kubernetes-sigs/node-feature-discovery/blob/7df0f17f6838183f67f5f6b662fb8715e598a7de/deployment/helm/node-feature-discovery/values.yaml#L97

Reply:
@fmuyassarov what do you suggest? Add all `livenessProbe` and `readinessProbe` settings in values.yaml? I don't think this makes sense as NFD meaningfully only supports this one type of health probe.

If we want to add configurability then I think it would be best to do in a separate PR, enabling nfd-master and nfd-worker in a similar way.

Reply:
> @fmuyassarov what do you suggest? Add all `livenessProbe` and `readinessProbe` settings in values.yaml? I don't think this makes sense as NFD meaningfully only supports this one type of health probe.

By one type, I guess you meant gRPC type probe. My point was that to add whatever is here as commented out (since we don't expect probes to be configured by users) in values.yaml for the sake of extra documentation. But it is not a big deal and can be exempt from adding.

Reply:
@fmuyassarov
Added with a comment, please take a look